snuba/admin/static/clickhouse_migrations/index.tsx

```diff
@@ -41,17 +43,16 @@ function ClickhouseMigrations(props: { api: Client }) {
       );
     }
     console.log("executing !", action);
+    executeRealRun(action);
```

Comment:
we need to add the `force` flag or else this will not run for real, the dry run ignores the need for the `force` flag but we'll need it for real for the following situations

* a migration is blocking (already has a confirm dialog)
* trying to reverse a migration that has already be run (needs the confirm dialog)

Reply:
fixed. added the force arg